udata_front/theme/gouvfr/templates/post/display.html

```diff
@@ -101,7 +101,11 @@ <h2 class="fr-h2">{{ _('Actions') }}</h2>
 
 {% if previous_post or next_post %}
 <section class="fr-container fr-my-7w fr-py-3w border-top">
-    <div class="fr-grid-row fr-grid-row--gutters justify-between">
+    <nav
+        role="navigation"
+        aria-label="{{_("Navigate in the page collection")}}"
```

Comment:
Don't forget to extract the translation ;)

Maybe it should be:
```suggestion
        aria-label="{{_("Navigate in the post collection")}}"
```